project_task_material_analytic_partner/README.rst

```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================
+Partner in task materials
+=========================
+
+This module adds the partner on task materials and propagate it to the
+auto-generated analytic entries on the "Other partner" field.
+
+Usage
+=====
+
+Go to Project > Tasks, and you will find the partner field in material lines.
+
+Using this partner, when you consume these materials, the target of the invoice
```

Comment:
Change this text to reflect better the purpose of the module (as in 10.0, there's no invoicing of analytic lines):
```
Using this partner, when you consume the materials, the other partner field in the generated analytic lines will be this one instead the one in the project.
```